starcheck/utils.py

```diff
@@ -543,3 +544,20 @@ def proseco_probs(**kw):
     ]
 
     return p_acqs, float(-np.log10(acq_cat.calc_p_safe())), float(np.sum(p_acqs))
+
+
+def vehicle_filter_backstop(backstop_file, outfile):
+    """
+    Filter the backstop file to remove SCS 131, 132, 133 except MP_OBSID commands.
+    This is basically equivalent to the vehicle backstop file, but the MP_OBSID
+    commands are useful for ACA to associate maneuvers with observations.
+    """
+    # Use parse_cm read_backstop_as_list instead of kadi.commands.read_backstop
+    # as we want the params to keep the SCS to write back later.
```

Comment:
@taldcroft as I was in a bit of a hurry I didn't dig much into this, but to me it made more sense to use the validated and tagged read-in backstop instead of a grep of some kind in the file.  But then if writing out a backstop, it seemed to make sense to use write_backstop... which seemed to have the params it wanted if I used parse_cm's own methods for reading backstop instead of the kadi versions.